### PR TITLE
Add signatures to logs

### PR DESCRIPTION
**Pull request by terencechain**
As we see more errors like "Signature did not verify" surges in testnet, it'll be really helpful to log these signatures for better debug

### CODE REVIEW

File: validator/client/validator_propose.go [hunk 1]
```diff
@@ -77,6 +77,7 @@ func (v *validator) ProposeBlock(ctx context.Context, slot uint64, pubKey [48]by
 		"blockRoot":       blkRoot,
 		"numAttestations": len(b.Body.Attestations),
 		"numDeposits":     len(b.Body.Deposits),
+		"signature":       fmt.Sprintf("%#x", bytesutil.Trunc(b.Signature)),
```
Comment (prestonvanloon) [inline validator/client/validator_propose.go]: How does a truncated signature help?

Comment (terencechain) [reply validator/client/validator_propose.go]: Use-case i was looking to is to match signature failure logs on beacon node with the proposing log/attesting log on the validator client

Comment (rauljordan) [reply validator/client/validator_propose.go]: I'd rather have the full signature as a debug log @terencechain what do you think?

Comment (terencechain) [reply validator/client/validator_propose.go]: That's fine too. Done!

**Pull request merged**